### ISSUE TITLE
Running error

### ISSUE THREAD

**Issue by SteveTsui** [ShaohuiLin/GAL#2]
Use  train_batch_size=64,pretrained model=resnet_56.pt. Command is ' python main.py --teacher_dir ...'. But got 'Warning: NaN or Inf found in input tensor.'. Can u offer some advice?

**Comment by SteveTsui** [comment]
If use train_batch_size=1, this problem will disappear. otherwise it occurs.

**Comment by colorjam** [comment]
> If use train_batch_size=1, this problem will disappear. otherwise it occurs.

Could you describe where the NaNs appear? We have not encountered this problem. We will update the code soon, maybe you can try the new ones.

**Comment by colorjam** [comment]
@SteveTsui  Could you describe where the NaNs appear? We have not encountered this problem. We will update the code soon, maybe you can try the new ones.

**Comment by SteveTsui** [comment]
I think it shows up at the backpropagation step. When I set the breakpoint at the end of train epoch(line 250, main.py), the problem doesn't show up.But it does show up when I set the breakpoint outside the train function(line 97, main.py). I am a student of one of the co-authors. Maybe u r a postgraduate of Xiamen University, I would appreciate it if u can contact me by WeChat. BTW, my WeChat ID is 'bookgoodroom'. Thx!
